secboot/secboot_hooks.go

```diff
@@ -195,11 +195,6 @@ func unlockVolumeUsingSealedKeyFDERevealKeyV2(sealedEncryptionKeyFile, sourceDev
 
 	// the output of fde-reveal-key is the unsealed key
 	options := activateVolOpts(opts.AllowRecoveryKey)
-	// FIXME: consider setting it in activateVolOpts if we keep
-	// this function separate from the tpm one for key data in
-	// files. Otherwise we should only set it if we provide key
-	// data generation 1.
-	options.Model = model
 
 	sbSetModel(model)
```

Comment:
btw. is this covered by existing tests?

Reply:
We do not verify it. I can add that in another PR.

Reply:
A followup PR is fine.

Reply:
It was small. So I added it.